src/Node/TaggedNodeTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Behat Gherkin Parser.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Gherkin\Node;
+
+/**
+ * This trait partially implements {@see TaggedNodeInterface}.
+ */
+trait TaggedNodeTrait
```

Comment:
I suggest tagging this trait as `@internal`. I don't think we should consider the trait itself as part of the public API covered by BC. It should stay an implementation detail of our classes

Reply:
I was totally wondering about that - good point.